src/plot_data.jl

```diff
@@ -212,7 +215,10 @@ function _filter_curtailment!(
     end
 end
 
-function get_generation_data(results::PSI.ProblemResults; kwargs...)
+function get_generation_data(
+    results::Union{PSI.SimulationProblemResults, PSI.ProblemResults};
```

Comment:
I think they have an abstract type `PSIResults` probably you don't need the union.